hermes-domain/src/main/java/eu/socialedge/hermes/domain/timetable/Schedule.java

```diff
@@ -76,6 +80,14 @@ public void scheduleAvailability(ScheduleAvailability scheduleAvailability) {
         this.scheduleAvailability = requireNotNull(scheduleAvailability);
     }
 
+    public String description() {
+        return description;
+    }
+
+    public void description(String description) {
+        this.description = requireNotBlank(description);
```

Comment:
It's a kind of schedule's name, used in Departures view, so it must be specified.
But from the domain perspective, it's not important too much.